topics/assembly/metadata.yaml

```diff
@@ -27,7 +27,7 @@ material:
     type: "tutorial"
     name: "general-introduction"
     zenodo_link: "https://doi.org/10.5281/zenodo.582600"
-    galaxy_tour: ""
+    galaxy_tour: "yes"
```

Comment:
If you use `yes` (or `no`) without the double quotes, it becomes a boolean. This allows:

- to be slightly more standard/interoperable
- to use conditions in the templates (`{% if tutorial.galaxy_tour %}`)